cocos/base/Utils.h

```diff
@@ -53,6 +54,9 @@ utils::nextPOT function is licensed under the same license that is used in Textu
 */
 
 namespace utils {
+
+CC_DLL std::string getStacktrace(uint skip = 0, uint maxDepth = static_cast<uint>(-1));
```

Comment:
It is better to use [UINT_MAX](https://en.cppreference.com/w/cpp/types/climits) instead of `maxDepth = static_cast<uint>(-1)`.